lib/amplitude.dart

```diff
@@ -270,7 +270,7 @@ class Amplitude {
 
   /// Flush events in storage.
   Future<void> flush() async {
-    await await _channel
+    await _channel
```

Comment:
Could we return this instead?
```suggestion
    return await _channel
```

Reply:
@chungdaniel aha good catch too 🤝